website/Rakefile

```diff
@@ -3,16 +3,18 @@ require 'html-proofer'
 require 'etc'
 
 task :test do
-  FileUtils.rm_rf('./.testcontent')
-  sh "bundle exec jekyll build --config _config.yml,_config_test.yml"
-  HTMLProofer.check_directory("./.testcontent", {
+  HTMLProofer.check_directory("./content", {
     :typhoeus => {
       :timeout => 60,
       :connecttimeout => 40 },
     :allow_hash_href => true,
     :check_html => true,
     :file_ignore => [/javadoc/, /v2/, /pydoc/],
     :url_ignore => [
+        # Javadocs and Pydocs are only available on asf-site branch
+        /documentation\/sdks\/javadoc/,
+        /documentation\/sdks\/pydoc/,
```

Comment:
Does this mean that we will need to check in the generated javadocs here (in the asf-site branch)?
If so:
 * We'll need to update the [release manager scripts](https://github.com/apache/beam/blob/master/release/src/main/scripts/build_release_candidate.sh). Maybe create a JIRA to track that?
 * Won't this explode the size of the repo? I know this has been discussed by others, and we can leave it for later, but how feasible is to avoid checking in generated files? (now or in the future)

Reply:
I'm not sure yet where the generated docs go, it's going to either be apache/beam-site or directly push to gitbox. TBD when I get responses from INFRA.

This bug tracks generation of java and pydocs: https://issues.apache.org/jira/browse/BEAM-4498